tiles/src/main/java/com/protomaps/basemap/layers/Buildings.java

```diff
@@ -50,8 +83,62 @@ public List<VectorTile.Feature> postProcess(int zoom, List<VectorTile.Feature> i
     }
     items = Area.filterArea(items, 0);
 
-    if (zoom >= 14)
+    // DEBUG
+    //items = Area.addAreaTag(items);
+
+    if (zoom >= 15)
       return items;
+
+    // quantize height by zoom when less than max_zoom 15 to facilitate better feature merging
```

Comment:
Buildings weren't really merging much before because of all the unique values. Quantizing them encourages more merges.